rotkehlchen/exchanges/exchange.py

```diff
@@ -53,7 +61,7 @@ def __init__(
             'secret for {} should be a bytestring'.format(name)
         )
         super().__init__()
-        self.name = name
+        self.   name = name
```

Comment:
this looks wrong. Didn't the linter complain?

Reply:
As I remember, no. I'll fix it